bedrock/mozorg/templates/mozorg/mpl/license-policy.html

```diff
@@ -15,14 +15,15 @@
 {% block content %}
 
 <div id="main-content">
-  <h1>Mozilla Foundation License Policy</h1>
+  <h1>Mozilla Source Code License Policy</h1>
   <p class="version"><em>Version 4.0</em></p>
 
   <section>
     <h2 id="Introduction">Introduction</h2>
     <p><em>Overview: <a href='{{ static("img/mozorg/mpl/license-policy-flowchart.png") }}'>this flowchart</a> provides a summary of the policy that can help you decide which license to use.</em></p>
-    <p>This policy determines which license to use for source files stored in Mozilla-controlled code repositories, whether hosted at Mozilla or elsewhere - "Mozilla Repositories". Committers to Mozilla Repositories must comply with this policy, as they agreed to do in the <a href="http://mozilla.org/hacking/committer/committers-agreement.pdf">Committer's Agreement</a>.</p>
-    <p>The following is a brief summary of the Mozilla license policy:</p>
+    <p>This policy determines which license to use for source files stored in Mozilla-controlled code repositories, whether hosted at Mozilla or elsewhere - "Mozilla Repositories". Committers to Mozilla Repositories must comply with this policy, as they agreed to do in the <a href="http://mozilla.org/hacking/committer/committers-agreement.pdf">Committer's Agreement</a>.
```

Comment:
Be good to go ahead and update this link to the new one and avoid some redirects for the user:

https://static.mozilla.com/foundation/documents/commit-access/committers-agreement.pdf